tutorials/io/30_reading_fnirs_data.py

```diff
@@ -25,7 +25,6 @@
              Manual modification of channel names and metadata
              is not recommended.
 
-
```

Comment:
Can you add a section to this document saying that MNE reorders the channels on import to be consistent across file types? Also describe the new ordering as different users from different systems will have different expectations.

Reply:
See suggestion above.